collectors/python.d.plugin/am2320/README.md

```diff
@@ -0,0 +1,37 @@
+# AM2320
+This module will display a graph of the temperture and humity from a AM2320 sensor
+
+**Requirements:**
+ - Adafruit Circuit Python AM2320 library
+ - Adafruit AM2320 I2C sensor
+ - Python 3
```

Comment:
The module needs to work with Python 2 as well as Python 3 if at all possible.  Your code looks like it should work find, but I'm not sure about the Adafruit libraries.